### PR TITLE
added async_req parameter (default to True) to ApiClient constructor

### PR DESCRIPTION
**Pull request by huang195**
Taking a stab at issue #471 
@dims ptal

### CODE REVIEW

File: kubernetes/client/api_client.py [hunk 1]
```diff
@@ -59,12 +59,14 @@ class ApiClient(object):
         'object': object,
     }
 
-    def __init__(self, configuration=None, header_name=None, header_value=None, cookie=None):
+    def __init__(self, configuration=None, header_name=None, header_value=None, cookie=None, async_req=True):
```
Comment (roycaihw) [inline kubernetes/client/api_client.py]: This file is auto-generated by [swagger-codegen](https://github.com/kubernetes-client/gen). You could contribute to swagger-codegen [python template](https://github.com/swagger-api/swagger-codegen/blob/d5f36191998396f4810155dd19cecdf697a72e95/modules/swagger-codegen/src/main/resources/python/api_client.mustache). When you did, we can import the changes here. Ref: https://github.com/kubernetes-client/python/issues/226

Comment (huang195) [reply kubernetes/client/api_client.py]: @roycaihw sure, can migrate the code change there, but other than that, the code looks ok?

Comment (roycaihw) [reply kubernetes/client/api_client.py]: the code change makes sense to me, although I'm not a python expert :)

Comment (dims) [reply kubernetes/client/api_client.py]: @huang195 looks good to me. please do follow the path mentioned by @roycaihw

Comment (huang195) [reply kubernetes/client/api_client.py]: I created the PR here: https://github.com/swagger-api/swagger-codegen/pull/7782

**Comment by roycaihw** [comment]
It looks like the upstream fix was stale. We recently bumped our dependency to use up-to-date upstream generator https://github.com/kubernetes-client/python/pull/931. If this issue has been fixed upstream, we should have picked the fix up

/close

**Pull request closed without merging**